index.js

```diff
@@ -20,6 +20,14 @@ function compile(data, opts) {
 		const filePath = file.path;
 		const env = (opts && opts.env) || new nunjucks.Environment(new nunjucks.FileSystemLoader(file.base), opts);
 
+		if (filters) {
+			for (const key in filters) {
```

Comment:
Use a `for-of` loop.

And use `Object.keys(filters)` and you can drop the hasOwnProp thing.

Reply:
Will do.